wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,332 @@
+| ![][flag_US] [idke](https://osu.ppy.sh/users/4650315) | 2019-06-07 | 2019-07-09 | [\[1\]](https://web.archive.org/web/20190607104421/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/idke/) [\[3\]](https://www.reddit.com/r/osugame/comments/bxs9zw/idke_claris_kimi_no_shiranai_monogatari_serenity/) [\[4\]](https://www.youtube.com/watch?v=WEszRyOKHlA) |
+| ![][flag_US] [Vaxei](https://osu.ppy.sh/users/4787150) | 2019-07-09 | 2019-07-11 | [\[1\]](https://web.archive.org/web/20190709104645/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/vaxei/) [\[3\]](https://www.pcgamer.com/two-teens-are-on-a-crazy-world-record-race-in-extremely-challenging-rhythm-game-osu/) [\[4\]](https://www.reddit.com/r/osugame/comments/caugmb/vaxei_is_now_officially_1_passing_idke/) [\[5\]](https://www.youtube.com/watch?v=7AYkfX2KZfs) |
+| ![][flag_US] [idke](https://osu.ppy.sh/users/4650315) | 2019-07-11 | 2019-07-23 | [\[1\]](https://web.archive.org/web/20190711164923/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/idke/) [\[3\]](https://www.pcgamer.com/two-teens-are-on-a-crazy-world-record-race-in-extremely-challenging-rhythm-game-osu/) [\[4\]](https://www.reddit.com/r/osugame/comments/cbwncx/idke_is_now_rank_1_passing_vaxei/) [\[5\]](https://www.youtube.com/watch?v=R5inaaITdT4) |
+| ![][flag_US] [Vaxei](https://osu.ppy.sh/users/4787150) | 2019-07-23 | 2019-10-06 | [\[1\]](https://web.archive.org/web/20190724021716/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/vaxei/) [\[3\]](https://www.youtube.com/watch?v=PVFcCcgwlwg) |
+| ![][flag_DE] [Whitecat](https://osu.ppy.sh/users/4504101) | 2019-10-06 | *Present* | [\[1\]](https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://osu.ppy.sh/users/4504101/) [\[3\]](https://ameobea.me/osutrack/user/whitecat/) [\[4\]](https://www.reddit.com/r/osugame/comments/de8duf/whitecat_is_now_1_global_on_osustandard/) [\[5\]](https://www.youtube.com/watch?v=Bx4R7lovF-0) |
+
+## 2020
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_DE] [Whitecat](https://osu.ppy.sh/users/4504101) | 2019-10-06 | *Present* | [\[1\]](https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://osu.ppy.sh/users/4504101/) [\[3\]](https://ameobea.me/osutrack/user/whitecat/) [\[4\]](https://www.reddit.com/r/osugame/comments/de8duf/whitecat_is_now_1_global_on_osustandard/) [\[5\]](https://www.youtube.com/watch?v=Bx4R7lovF-0) |
+
+## 2021
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_DE] [Whitecat](https://osu.ppy.sh/users/4504101) | 2019-10-06 | *Present* | [\[1\]](https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://osu.ppy.sh/users/4504101/) [\[3\]](https://ameobea.me/osutrack/user/whitecat/) [\[4\]](https://www.reddit.com/r/osugame/comments/de8duf/whitecat_is_now_1_global_on_osustandard/) [\[5\]](https://www.youtube.com/watch?v=Bx4R7lovF-0) |
```

Comment:
```suggestion
| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 2014-08-17 | *2015-02-06* | [\[1\]](https://web.archive.org/web/20140818131806/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/hvick225) [\[3\]](https://osu.ppy.sh/community/forums/topics/235678) [\[4\]](https://www.youtube.com/watch?v=4jW75bKPVP0) [\[5\]](https://osu.ppy.sh/scores/osu/1782720884) |

## 2015

| Player | From | To | Sources |
| --: | :-- | :-- | :-- |
| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | *2014-08-17* | 2015-02-06 | [\[1\]](https://web.archive.org/web/20140818131806/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/hvick225) [\[3\]](https://osu.ppy.sh/community/forums/topics/235678) [\[4\]](https://www.youtube.com/watch?v=4jW75bKPVP0) [\[5\]](https://osu.ppy.sh/scores/osu/1782720884) |
| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 2015-02-06 | 2015-03-25 | [\[1\]](https://web.archive.org/web/20150206190712/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/wubwoofwolf) [\[3\]](https://old.reddit.com/r/osugame/comments/2v0gqj/www_is_now_1/) |
| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 2015-03-25 | 2015-11-22 | [\[1\]](https://web.archive.org/web/20150325184217/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/hvick225) [\[3\]](https://old.reddit.com/r/osugame/comments/308mes/hvick_takes_1_with_560pp_play/) [\[4\]](https://www.youtube.com/watch?v=DHhBkdCLizs) [\[5\]](https://osu.ppy.sh/scores/osu/1906575547) |
| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 2015-11-22 | 2015-11-28 | [\[1\]](https://web.archive.org/web/20151122193734/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/rafis) [\[3\]](https://old.reddit.com/r/osugame/comments/3tufa4/rafis_is_now_the_rank_1_osustandard_player/) [\[4\]](https://www.youtube.com/watch?v=2elafCS4CBA) [\[5\]](https://osu.ppy.sh/scores/osu/2017650172) |
| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 2015-11-28 | 2015-12-17 | [\[1\]](https://ameobea.me/osutrack/user/hvick225) [\[2\]](https://old.reddit.com/r/osugame/comments/3um0me/hvick_is_back_to_1/) [\[3\]](https://www.youtube.com/watch?v=vF2ic2treck) [\[4\]](https://osu.ppy.sh/scores/osu/2020492666) |
| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 2015-12-17 | 2015-12-19 | [\[1\]](https://web.archive.org/web/20151217200752/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/rafis) [\[3\]](https://old.reddit.com/r/osugame/comments/3x8i31/rafis_is_1_again/) [\[4\]](https://www.youtube.com/watch?v=dqTxJhPRuh0) |
| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 2015-12-19 | 2015-12-30 | [\[1\]](https://web.archive.org/web/20151219205754/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/hvick225) [\[3\]](https://old.reddit.com/r/osugame/comments/3xgpdg/hvick225_1_yet_again/) [\[4\]](https://www.youtube.com/watch?v=B-THPoG_Kd0) [\[5\]](https://osu.ppy.sh/scores/osu/2031299587) |
| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 2015-12-30 | 2015-12-31 | [\[1\]](https://web.archive.org/web/20151230231205/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/rafis) [\[3\]](https://old.reddit.com/r/osugame/comments/3ysy54/rafis_is_1_once_again/) [\[4\]](https://www.youtube.com/watch?v=Gr-DC2dH_Sw) [\[5\]](https://osu.ppy.sh/scores/osu/2038079133) |
| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 2015-12-31 | *2016-05-12* | [\[1\]](https://web.archive.org/web/20160101024536/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/hvick225) [\[3\]](https://old.reddit.com/r/osugame/comments/3ywmnf/hvick_now_1_again/) [\[4\]](https://osu.ppy.sh/scores/osu/2038565115) |

## 2016

| Player | From | To | Sources |
| --: | :-- | :-- | :-- |
| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | *2015-12-31* | 2016-05-12 | [\[1\]](https://web.archive.org/web/20160101024536/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/hvick225) [\[3\]](https://old.reddit.com/r/osugame/comments/3ywmnf/hvick_now_1_again/) [\[4\]](https://osu.ppy.sh/scores/osu/2038565115) |
| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 2016-05-12 | 2016-06-25 | [\[1\]](https://web.archive.org/web/20160512164834/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/rafis) [\[3\]](https://old.reddit.com/r/osugame/comments/4j0q5l/rafis_submitted_a_score_got_1_global/) |
| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2016-06-25 | 2016-06-27 | [\[1\]](https://web.archive.org/web/20160625160832/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/chocomint) [\[3\]](https://old.reddit.com/r/osugame/comments/4pstk8/cookiezi_is_now_1/) [\[4\]](https://www.youtube.com/watch?v=P2Ylc4oz1w0) |
| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 2016-06-27 | 2016-09-20 | [\[1\]](https://web.archive.org/web/20160627192658/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/rafis) [\[3\]](https://old.reddit.com/r/osugame/comments/4q4jdc/rafis_is_now_1/) [\[4\]](https://www.youtube.com/watch?v=BVGREMxrelg) |
| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2016-09-20 | 2016-09-25 | [\[1\]](https://web.archive.org/web/20160920022127/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/chocomint) [\[3\]](https://old.reddit.com/r/osugame/comments/53l45k/cookiezi_has_overtaken_rafis_as_rank_1/) [\[4\]](https://www.youtube.com/watch?v=VsYLIg67pYo) |
| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 2016-09-25 | 2016-10-03 | [\[1\]](https://web.archive.org/web/20160925212108/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/rafis) [\[3\]](https://old.reddit.com/r/osugame/comments/54ghvx/rafis_back_at_1/) [\[4\]](https://www.youtube.com/watch?v=BtSDzTScjbk) |
| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2016-10-03 | 2016-10-04 | [\[1\]](https://web.archive.org/web/20161004002312/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/chocomint) [\[3\]](https://old.reddit.com/r/osugame/comments/55qpfb/cookiezi_has_overtaken_rafis_as_1_again/) [\[4\]](https://www.youtube.com/watch?v=un2rRfZ2VSM) |
| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 2016-10-04 | 2016-10-08 | [\[1\]](https://web.archive.org/web/20161006200257/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/rafis) [\[3\]](https://old.reddit.com/r/osugame/comments/55ukyl/rafis_claimed_back_rank_1/) [\[4\]](https://osu.ppy.sh/scores/osu/2184102984) |
| ![][flag_KR] [firebat92](https://osu.ppy.sh/users/1777162) | 2016-10-08 | 2016-10-10 | [\[1\]](https://web.archive.org/web/20161008045636/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/firebat92) [\[3\]](https://old.reddit.com/r/osugame/comments/56fg8r/angelmegumin_is_now_1_global/) [\[4\]](https://www.youtube.com/watch?v=nL5ZQgBMWtM) |
| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 2016-10-10 | 2016-10-10 | [\[1\]](https://ameobea.me/osutrack/user/rafis) [\[2\]](https://old.reddit.com/r/osugame/comments/56sl1o/rafis_takes_1_back_by_01pp/) [\[4\]](https://www.youtube.com/watch?v=b-__ryiioI4) |
| ![][flag_KR] [firebat92](https://osu.ppy.sh/users/1777162) | 2016-10-10 | 2016-10-10 | [\[1\]](https://ameobea.me/osutrack/user/firebat92) [\[2\]](https://old.reddit.com/r/osugame/comments/56syc4/angelmegumin_is_back_at_1/) [\[3\]](https://www.youtube.com/watch?v=VcHXexiJp0c) |
| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 2016-10-10 | 2016-10-11 | [\[1\]](https://ameobea.me/osutrack/user/rafis) [\[2\]](https://old.reddit.com/r/osugame/comments/56t713/rafis_takes_1_back_again/) [\[3\]](https://www.youtube.com/watch?v=7cKdrio81Nc) |
| ![][flag_KR] [firebat92](https://osu.ppy.sh/users/1777162) | 2016-10-11 | 2016-10-18 | [\[1\]](https://web.archive.org/web/20161011005428/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/firebat92) |
| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2016-10-18 | 2016-10-20 | [\[1\]](https://web.archive.org/web/20161018110255/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/chocomint) |
| ![][flag_KR] [firebat92](https://osu.ppy.sh/users/1777162) | 2016-10-20 | 2016-10-21 | [\[1\]](https://old.reddit.com/r/osugame/comments/58gmoh/angelsims_back_to_1/) |
| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2016-10-21 | 2016-10-21 |  [\[1\]](https://web.archive.org/web/20161021090628/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/chocomint) |
| ![][flag_KR] [firebat92](https://osu.ppy.sh/users/1777162) | 2016-10-21 | 2016-10-23 | [\[1\]](https://web.archive.org/web/20161021202725/https://osu.ppy.sh/p/pp) [\[2\]](https://old.reddit.com/r/osugame/comments/58muhj/angelsim_is_now_1_again/) |
| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2016-10-23 | *2018-01-21* | [\[1\]](https://web.archive.org/web/20161023123340/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/chocomint) [\[3\]](https://old.reddit.com/r/osugame/comments/58xa3r/cookiezi_is_now_1_again/) [\[4\]](https://www.youtube.com/watch?v=tnNHXFKe38c) |

## 2017

| Player | From | To | Sources |
| --: | :-- | :-- | :-- |
| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | *2016-10-23* | *2018-01-21* | [\[1\]](https://web.archive.org/web/20161023123340/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/chocomint) [\[3\]](https://old.reddit.com/r/osugame/comments/58xa3r/cookiezi_is_now_1_again/) [\[4\]](https://www.youtube.com/watch?v=tnNHXFKe38c) |

## 2018

| Player | From | To | Sources |
| --: | :-- | :-- | :-- |
| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | *2016-10-23* | 2018-01-21 | [\[1\]](https://web.archive.org/web/20161023123340/https://osu.ppy.sh/p/pp) [\[2\]](https://ameobea.me/osutrack/user/chocomint) [\[3\]](https://old.reddit.com/r/osugame/comments/58xa3r/cookiezi_is_now_1_again/) [\[4\]](https://www.youtube.com/watch?v=tnNHXFKe38c) |
| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 2018-01-21 | 2018-04-15 | [\[1\]](https://web.archive.org/web/20180122113018/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/rafis) [\[3\]](https://old.reddit.com/r/osugame/comments/7s1i95/rafis_is_now_1_in_osustd/) [\[4\]](https://www.youtube.com/watch?v=M6SPs67ZPGw) |
| ![][flag_CL] [Mathi](https://osu.ppy.sh/users/5339515) | 2018-04-15 | 2018-04-22 | [\[1\]](https://web.archive.org/web/20180420034757/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/mathi) [\[3\]](https://old.reddit.com/r/osugame/comments/8ciyxi/mathi_is_now_1_global/) [\[4\]](https://www.youtube.com/watch?v=p-DE64dd-VI) |
| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 2018-04-22 | 2018-04-25 | [\[1\]](https://ameobea.me/osutrack/user/rafis) [\[2\]](https://old.reddit.com/r/osugame/comments/8e04aw/rafis_has_taken_back_1/) [\[3\]](https://www.youtube.com/watch?v=2uMxhsFiw54) |
| ![][flag_CL] [Mathi](https://osu.ppy.sh/users/5339515) | 2018-04-25 | 2018-05-14 | [\[1\]](https://web.archive.org/web/20180429184016/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/mathi) [\[3\]](https://old.reddit.com/r/osugame/comments/8ewq08/mathi_is_1_global_again/) [\[4\]](https://www.youtube.com/watch?v=4JpMs_ZeQRM) |
| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2018-05-14 | 2018-05-28 | [\[1\]](https://ameobea.me/osutrack/user/chocomint) [\[2\]](https://old.reddit.com/r/osugame/comments/8jgje8/cookiezi_is_now_global_1/) |
| ![][flag_CL] [Mathi](https://osu.ppy.sh/users/5339515) | 2018-05-28 | 2018-08-10 | [\[1\]](https://ameobea.me/osutrack/user/mathi) [\[2\]](https://old.reddit.com/r/osugame/comments/8mlo38/mathi_is_now_1_global/) |
| ![][flag_KR] [firebat92](https://osu.ppy.sh/users/1777162) | 2018-08-10 | 2018-09-02 | [\[1\]](https://web.archive.org/web/20180810183300/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/firebat92) [\[3\]](https://old.reddit.com/r/osugame/comments/965gv5/firebat92_is_now_1_global_passing_mathi/) |
| ![][flag_CL] [Mathi](https://osu.ppy.sh/users/5339515) | 2018-09-02 | 2018-11-24 | [\[1\]](https://web.archive.org/web/20180904171717/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/mathi) [\[3\]](https://old.reddit.com/r/osugame/comments/9ca4ew/mathi_is_now_1/) |
| ![][flag_US] [idke](https://osu.ppy.sh/users/4650315) | 2018-11-24 | 2018-12-10 | [\[1\]](https://web.archive.org/web/20181124210243/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/idke) [\[3\]](https://old.reddit.com/r/osugame/comments/a01mag/idke_has_reached_rank_1/) [\[4\]](https://www.youtube.com/watch?v=rYnCHFDUKR8) |
| ![][flag_CL] [Mathi](https://osu.ppy.sh/users/5339515) | 2018-12-10 | 2018-12-11 | [\[1\]](https://ameobea.me/osutrack/user/mathi) [\[2\]](https://old.reddit.com/r/osugame/comments/a4z4av/osustd_mathi_is_now_1_global_passing_idke/) |
| ![][flag_US] [idke](https://osu.ppy.sh/users/4650315) | 2018-12-11 | 2018-12-13 | [\[1\]](https://ameobea.me/osutrack/user/idke) [\[2\]](https://old.reddit.com/r/osugame/comments/a53xo2/idke_is_now_1_global_with_mouse_only_and_his_head/) |
| ![][flag_CL] [Mathi](https://osu.ppy.sh/users/5339515) | 2018-12-13 | 2018-12-16 | [\[1\]](https://web.archive.org/web/20181214084650/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/mathi) [\[3\]](https://old.reddit.com/r/osugame/comments/a5q7i3/mathi_is_now_1_global/) |
| ![][flag_US] [idke](https://osu.ppy.sh/users/4650315) | 2018-12-16 | 2018-12-17 | [\[1\]](https://ameobea.me/osutrack/user/idke) [\[2\]](https://old.reddit.com/r/osugame/comments/a6taiu/idke_is_1_global_again_overtaking_mathi/) |
| ![][flag_CL] [Mathi](https://osu.ppy.sh/users/5339515) | 2018-12-17 | *2019-01-12* | [\[1\]](https://web.archive.org/web/20181218083408/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/mathi) [\[3\]](https://old.reddit.com/r/osugame/comments/a6xerb/mathi_is_1_again/) [\[4\]](https://www.youtube.com/watch?v=PA3zCdb052c) |

## 2019

| Player | From | To | Sources |
| --: | :-- | :-- | :-- |
| ![][flag_CL] [Mathi](https://osu.ppy.sh/users/5339515) | *2018-12-17* | 2019-01-12 | [\[1\]](https://web.archive.org/web/20181218083408/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/mathi) [\[3\]](https://old.reddit.com/r/osugame/comments/a6xerb/mathi_is_1_again/) [\[4\]](https://www.youtube.com/watch?v=PA3zCdb052c) |
| ![][flag_US] [idke](https://osu.ppy.sh/users/4650315) | 2019-01-12 | 2019-02-11 | [\[1\]](https://web.archive.org/web/20190117233225/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/idke/) [\[3\]](https://old.reddit.com/r/osugame/comments/af1x9g/idke_is_now_1_global_again_passing_mathi/) [\[4\]](https://www.youtube.com/watch?v=X3F2y-WQ3vM) |
| ![][flag_KR] [FlyingTuna](https://osu.ppy.sh/users/9224078) | 2019-02-11 | 2019-02-17 | [\[1\]](https://web.archive.org/web/20190211175218/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/flyingtuna) [\[3\]](https://old.reddit.com/r/osugame/comments/apgi87/flyingtuna_has_reached_1/) [\[4\]](https://www.youtube.com/watch?v=OEVTkmKYYPY) |
| ![][flag_US] [idke](https://osu.ppy.sh/users/4650315) | 2019-02-17 | 2019-02-28 | [\[1\]](https://web.archive.org/web/20190217221833/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/idke/) [\[3\]](https://old.reddit.com/r/osugame/comments/arnqwe/idke_is_now_1_global/) [\[4\]](https://www.youtube.com/watch?v=XgkmFxuaqVQ) |
| ![][flag_KR] [FlyingTuna](https://osu.ppy.sh/users/9224078) | 2019-02-28 | 2019-03-04 | [\[1\]](https://web.archive.org/web/20190228121742/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/flyingtuna) [\[3\]](https://old.reddit.com/r/osugame/comments/avgo0d/flyingtuna_is_now_1_with_a_900pp_score/) [\[4\]](https://www.youtube.com/watch?v=68ESrkRnEes) |
| ![][flag_US] [idke](https://osu.ppy.sh/users/4650315) | 2019-03-04 | 2019-03-29 | [\[1\]](https://web.archive.org/web/20190305084348/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/idke/) [\[3\]](https://old.reddit.com/r/osugame/comments/ax70h4/idke_is_now_1_global/) [\[4\]](https://www.youtube.com/watch?v=8lqdzflPVq4) |
| ![][flag_KR] [FlyingTuna](https://osu.ppy.sh/users/9224078) | 2019-03-29 | 2019-03-31 | [\[1\]](https://web.archive.org/web/20190330150654/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/flyingtuna) [\[3\]](https://old.reddit.com/r/osugame/comments/b6zmvd/flyingtuna_is_now_1_global/) [\[4\]](https://www.youtube.com/watch?v=rVv0bvV62rU) |
| ![][flag_US] [Freddie Benson](https://osu.ppy.sh/users/7342622) | 2019-03-31 | 2019-03-31 | [\[1\]](https://web.archive.org/web/20190331101522/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/freddie%20benson/) [\[3\]](https://old.reddit.com/r/osugame/comments/b7iiem/klug_is_now_1_global/) [\[4\]](https://www.youtube.com/watch?v=60RMSzBiMnY) |
| ![][flag_KR] [FlyingTuna](https://osu.ppy.sh/users/9224078) | 2019-03-31 | 2019-05-08 | [\[1\]](https://web.archive.org/web/20190331232209/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/flyingtuna) [\[3\]](https://old.reddit.com/r/osugame/comments/b7oljb/walkingtuna_formerly_known_as_flyingtuna_is_now/) [\[4\]](https://www.youtube.com/watch?v=tU6k9nbg9hc) |
| ![][flag_US] [Freddie Benson](https://osu.ppy.sh/users/7342622) | 2019-05-08 | 2019-05-16 | [\[1\]](https://web.archive.org/web/20190509091205/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/freddie%20benson/) [\[3\]](https://old.reddit.com/r/osugame/comments/bmb7tp/klug_is_now_1_passing_walkingtuna/) [\[4\]](https://www.youtube.com/watch?v=y8vf_mznARM) |
| ![][flag_US] [idke](https://osu.ppy.sh/users/4650315) | 2019-05-16 | 2019-05-18 | [\[1\]](https://ameobea.me/osutrack/user/idke/) [\[2\]](https://old.reddit.com/r/osugame/comments/bpdrbk/idke_is_now_1_global_passing_freddie_benson/) [\[3\]](https://www.youtube.com/watch?v=S5yhTxznHIY&feature=youtu.be&ab_channel=cpol) |
| ![][flag_KR] [FlyingTuna](https://osu.ppy.sh/users/9224078) | 2019-05-18 | 2019-05-27 | [\[1\]](https://web.archive.org/web/20190518192017/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/flyingtuna) [\[3\]](https://old.reddit.com/r/osugame/comments/bq4yrb/walkingtuna_is_now_1_passing_idke/) [\[4\]](https://www.youtube.com/watch?v=RjJrpk5_XA8) |
| ![][flag_US] [Freddie Benson](https://osu.ppy.sh/users/7342622) | 2019-05-27 | 2019-05-31 | [\[1\]](https://web.archive.org/web/20190528114141/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/freddie%20benson/) [\[3\]](https://old.reddit.com/r/osugame/comments/btq1dv/freddie_benson_is_now_1/) [\[4\]](https://www.youtube.com/watch?v=6sk_b_7Bf7g) |
| ![][flag_US] [idke](https://osu.ppy.sh/users/4650315) | 2019-05-31 | 2019-06-06 | [\[1\]](https://web.archive.org/web/20190604130007/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/idke/) [\[3\]](https://old.reddit.com/r/osugame/comments/bv3keb/idke_is_now_1/) [\[4\]](https://www.youtube.com/watch?v=dT-9WG8hajk) |
| ![][flag_US] [Freddie Benson](https://osu.ppy.sh/users/7342622) | 2019-06-06 | 2019-06-07 | [\[1\]](https://web.archive.org/web/20190606065349/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/freddie%20benson/) [\[3\]](https://old.reddit.com/r/osugame/comments/bxc12c/freddie_benson_retakes_1/) [\[4\]](https://www.youtube.com/watch?v=WGlB23TXTjI) |
| ![][flag_US] [idke](https://osu.ppy.sh/users/4650315) | 2019-06-07 | 2019-07-09 | [\[1\]](https://web.archive.org/web/20190607104421/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/idke/) [\[3\]](https://old.reddit.com/r/osugame/comments/bxs9zw/idke_claris_kimi_no_shiranai_monogatari_serenity/) [\[4\]](https://www.youtube.com/watch?v=WEszRyOKHlA) |
| ![][flag_US] [Vaxei](https://osu.ppy.sh/users/4787150) | 2019-07-09 | 2019-07-11 | [\[1\]](https://web.archive.org/web/20190709104645/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/vaxei/) [\[3\]](https://www.pcgamer.com/two-teens-are-on-a-crazy-world-record-race-in-extremely-challenging-rhythm-game-osu/) [\[4\]](https://old.reddit.com/r/osugame/comments/caugmb/vaxei_is_now_officially_1_passing_idke/) [\[5\]](https://www.youtube.com/watch?v=7AYkfX2KZfs) |
| ![][flag_US] [idke](https://osu.ppy.sh/users/4650315) | 2019-07-11 | 2019-07-23 | [\[1\]](https://web.archive.org/web/20190711164923/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/idke/) [\[3\]](https://www.pcgamer.com/two-teens-are-on-a-crazy-world-record-race-in-extremely-challenging-rhythm-game-osu/) [\[4\]](https://old.reddit.com/r/osugame/comments/cbwncx/idke_is_now_rank_1_passing_vaxei/) [\[5\]](https://www.youtube.com/watch?v=R5inaaITdT4) |
| ![][flag_US] [Vaxei](https://osu.ppy.sh/users/4787150) | 2019-07-23 | 2019-10-06 | [\[1\]](https://web.archive.org/web/20190724021716/https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://ameobea.me/osutrack/user/vaxei/) [\[3\]](https://www.youtube.com/watch?v=PVFcCcgwlwg) |
| ![][flag_DE] [Whitecat](https://osu.ppy.sh/users/4504101) | 2019-10-06 | *Present* | [\[1\]](https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://osu.ppy.sh/users/4504101/) [\[3\]](https://ameobea.me/osutrack/user/whitecat/) [\[4\]](https://old.reddit.com/r/osugame/comments/de8duf/whitecat_is_now_1_global_on_osustandard/) [\[5\]](https://www.youtube.com/watch?v=Bx4R7lovF-0) |

## 2020

| Player | From | To | Sources |
| --: | :-- | :-- | :-- |
| ![][flag_DE] [Whitecat](https://osu.ppy.sh/users/4504101) | 2019-10-06 | *Present* | [\[1\]](https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://osu.ppy.sh/users/4504101/) [\[3\]](https://ameobea.me/osutrack/user/whitecat/) [\[4\]](https://old.reddit.com/r/osugame/comments/de8duf/whitecat_is_now_1_global_on_osustandard/) [\[5\]](https://www.youtube.com/watch?v=Bx4R7lovF-0) |

## 2021

| Player | From | To | Sources |
| --: | :-- | :-- | :-- |
| ![][flag_DE] [Whitecat](https://osu.ppy.sh/users/4504101) | 2019-10-06 | *Present* | [\[1\]](https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://osu.ppy.sh/users/4504101/) [\[3\]](https://ameobea.me/osutrack/user/whitecat/) [\[4\]](https://old.reddit.com/r/osugame/comments/de8duf/whitecat_is_now_1_global_on_osustandard/) [\[5\]](https://www.youtube.com/watch?v=Bx4R7lovF-0) |
```

- Replaced all www.reddit.com links with old.reddit.com.

- Archived all the old.reddit.com links used (I will get to the rest of the links momentarily).

- Removed a duplicate Cookiezi reign in 2016 (2016-10-23 | 2018-01-21)